src/background.js

```diff
@@ -62,6 +62,8 @@ const WITNET_FILE_NAME = {
 const WITNET_CONFIG_FILE_NAME = 'witnet.toml'
 const LATEST_RELEASES_URL =
   'https://api.github.com/repos/witnet/witnet-rust/releases/latest'
+const LATEST_RELEASE_COMPATIBLE =
```

Comment:
This is not the lastest release compatible, if witnet-rust publishes a patch release it will be the latest
```suggestion
const COMPATIBLE_RELEASE =
```